docbook/Admin_Guide/en-US/config/api.xml

```diff
@@ -35,11 +44,17 @@
 				specify reporter when adding issues or issue notes. Defaults to DEVELOPER.</para>
 			</listitem>
 		</varlistentry>
+		<varlistentry>
+			<term>$g_webservice_rest_enabled</term>
+			<listitem>
+				<para>Whether the REST API (experimental) is enabled or not.  Default OFF.</para>
+			</listitem>
+		</varlistentry>
 	</variablelist>
 
-	<section id="admin.config.soap.disable">
-		<title>Disabling the SOAP API</title>
-		<para>If you wish to temporarily disable the SOAP API it is sufficient to set the specific access thresholds to NOBODY:</para>
+	<section id="admin.config.api.disable">
+		<title>Disabling the webservice API</title>
+		<para>If you wish to temporarily disable the webservice API it is sufficient to set the specific access thresholds to NOBODY:</para>
 		<para><literal>$g_webservice_readonly_access_level_threshold = $g_webservice_readwrite_access_level_threshold = $g_webservice_admin_access_level_threshold = NOBODY;</literal></para>
 		<para>While the SOAP API will still be accessible, it will not allow users to retrieve or modify data.</para>
```

Comment:
> Does that effectively disable both APIs or just SOAP ?

At the moment, this applies to both REST/SOAP calls.

> Do we need / want distinct access levels for REST and SOAP ?

If we keep the concept, then we should keep it consistent across REST and SOAP.  But I'm considering removing this concept in favor of an ON/OFF for API and per feature authorization rules.  Note that with the REST API, I added a config option for ON/OFF which applies only to REST API.  This is useful to keep REST API disabled while it is in preview/beta.  It may be useful in the future, if a user wants to disable MantisBT API, but it will be switched from OFF to ON as the API gets out of preview/beta.